CMakeLists.txt

```diff
@@ -169,7 +169,10 @@ ENDIF(BUILD_PYTHON_INTERFACE)
 IF(BUILD_WITH_HPP_FCL_SUPPORT)
   ADD_DEFINITIONS(-DPINOCCHIO_WITH_HPP_FCL)
   LIST(APPEND CFLAGS_DEPENDENCIES "-DPINOCCHIO_WITH_HPP_FCL")
-  ADD_PROJECT_DEPENDENCY(hpp-fcl 1.7.3 REQUIRED PKG_CONFIG_REQUIRES "hpp-fcl >= 1.7.3")
+  ADD_PROJECT_DEPENDENCY(hpp-fcl REQUIRED PKG_CONFIG_REQUIRES "hpp-fcl >= 1.7.3")
+  IF(${hpp-fcl_VERSION} VERSION_LESS "1.7.3")
```

Comment:
I don’t understand why you need this change? ADD_PROJECT_DEPENDENCY should handle it internally. @nim65s could you confirm?

Reply:
CMake version checks are really boring on this topic. If you require X.Y.Z, and you have version A.B.C, it checks for `A == X && (B > Y || (B == Y && C >= Z))`.

I couldn't find a better alternative than the one @wxmerkt is using here when we released example-robot-data v4 and it broke everything just because 4 > 3, but I think we'll have to find a better solution at some point.

Reply:
Ok, I found the issue: https://github.com/jrl-umi3218/jrl-cmakemodules/blob/412628866e3613387c58e0961a05ce242dc93993/package-config.cmake#L167

Reply:
Nice catch

Reply:
So, we need to remove it. No?

Reply:
I've updated the cmake module to account for that properly. It will require an update of hpp-fcl with a new release (on going).